releases/release-1.32/release-notes/release-notes-draft.md

```diff
@@ -19,37 +12,44 @@
 
 ### API Change
 
+- ([#126620](https://github.com/kubernetes/kubernetes/pull/126620), [@yunwang0911](https://github.com/yunwang0911))
+- ACTION REQUIRED for custom scheduler plugin developers:
```

Comment:
should we do like https://github.com/kubernetes/kubernetes/pull/129126#discussion_r1877954146 ? (bold ACTION REQUIRED)